README

```diff
@@ -1 +1,67 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Java 8, available at (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+Maven 3+, available at (https://maven.apache.org/download.cgi)
+Python 2.4+, available at (https://www.python.org/download/releases/2.4/)
+
+### Installing
+
+Clone the full distribution from (https://github.com/prestodb/presto-manager)
+
+```
+git clone https://github.com/prestodb/presto-manager.git
+```
+Go the the directory you cloned the distribution
+
+```
+cd /directory/of/the/clone/presto-manager
+```
+Now you should be on the top level of `presto-manager` directory
+
+
+## Compiling and starting
+
+Use Maven to compile and generate executable JAR for the project
+
+```
+mvn clean install
+```
+
+Then `cd` into ./presto-manager-agent/target and ./presto-manager-controller/target, respectively.
```

Comment:
Instead of giving instructions to change directory, why not give instructions to use the full path to the executable from the project root?

(And say "change directory" instead of "`cd`").